main.cpp

```diff
@@ -73,12 +24,26 @@ void testPairToNumber(
     assert(pairNumber == expectedPairNumber);
 }
 
+std::string ToString()
+{
+    std::string codeToColorsMappings = "";
+    for(int pairNumber = 1; pairNumber <= 25; pairNumber++) {
```

Comment:
Do not hardcode the loop limit. It will be hard to remember to update this when a color is added or removed.

Reply:
I have updated the code. I created 2 functions, which return number of major and minor colors. In my ToString() function I have multiplied the returned values to get the count.